src/main/java/net/minecraftforge/client/event/RenderNameplateEvent.java

```diff
@@ -32,33 +33,41 @@
  * {@link #nameplateContent} contains the content being rendered on the name plate/tag. This can be changed by mods.<br>
  * {@link #originalContent} contains the original content being rendered on the name plate/tag. This cannot be
  * changed by mods.<br>
+ * {@link #entityRenderer} contains the entity renderer instance that renders the name plate/tag. This cannot be
+ * changed by mods.<br>
  * {@link #matrixStack} contains the matrix stack instance involved in rendering the name plate/tag. This cannot
  * be changed by mods.<br>
  * {@link #renderTypeBuffer} contains the render type buffer instance involved in rendering the name plate/tag.
  * This cannot be changed by mods.<br>
+ * {@link #packedLight} contains the sky and block light values used in rendering the name plate/tag.<br>
  * <br>
  * This event has a result. {@link HasResult}. <br>
  * ALLOW will force-render name plate/tag, DEFAULT will ignore the hook and continue using the vanilla check
  * & DENY will prevent name plate/tag from rendering<br>
  * <br>
  * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ * @param <T> the entity type of the renderer
  **/
 @Event.HasResult
-public class RenderNameplateEvent extends EntityEvent
+public class RenderNameplateEvent<T extends Entity> extends EntityEvent
```

Comment:
Type parameters on events are meaningless, unless `IGenericEvent` is used, which provides very limited filtering and is not applicable here.

Reply:
I recall it wasn't useful in RenderLivingEvent.Specials.Pre either. Shall I strip it from the class and @SuppressWarnings("rawtypes") instead?